examples/examples-gpu/nyc-taxi/rf-rapids.ipynb

```diff
@@ -4,47 +4,98 @@
    "cell_type": "markdown",
```

Comment:
<p>small note here, I think it should say the "RAPIDS docs" instead of "NVIDIA docs". NVIDIA likes to refer to RAPIDS as a separate thing from NVIDIA</p><p> </p><p>also would be nice to get nvdashboard in the saturngpu images, so then people can use the jupyter widget for it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>